util/sorc/compile_gfs_util_jet.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
What part of the workflow runs this on Jet?
Is this necessary for research purposes?
This seems like an operational product.

Reply:
gfs_util is not been used in JET fcst only run. It is just a place holder here to complete the build without error. If we are sure JET will never need the gfs_util, I can remove it. However, the build script will need to be modified to filter out JET target.

Reply:
Please delete this file.
We will revisit this when there is a need.

Reply:
Modified build_gfs_util.sh to filter out JET for now.

Reply:
This script is called `compile_gfs_util_jet.sh` and yet it has targets for `wcoss`, `wcoss_cray`, `hera` etc.

Reply:
compile_gfs_util_jet.sh is deleted